### PR TITLE
Rule fixes for Do you use IApiMarker with WebApplicationFactory?

### PR DESCRIPTION
**Pull request by DevStarOps**
Tasks from @tiagov8 

- [x] Please update the URL to *not* include “do you” as per https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Create-Rules 
See more info: https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Edit-Rules#renaming-rules 
- [x] Make all headings consistent (H3)
- [x] Fix the images Markdown syntax (We don’t add the emojis manually)
- [x] Include the prefixes “Figure: Good example - ” and “Figure: Bad example - " to all captions
- [x] Add me as the reviewer of your PRs

Tasks from @leehawkinsssw 

- [x] Review the SSW Rules categories under “Software Development” to see if there is a better fit for your www.ssw.com.au/rules/do-you-use-iapimarker-with-webapplicationfactory rule (perhaps https://ssw.com.au/rules/rules-to-better-unit-tests?)

### CODE REVIEW

File: rules/use-iapimarker-with-webapplicationfactory/rule.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 type: rule
 title: Do you use IApiMarker with WebApplicationFactory?
-uri: do-you-use-iapimarker-with-webapplicationfactory
```
Comment (tiagov8) [inline rules/use-iapimarker-with-webapplicationfactory/rule.md:4]: Hey @DevStarOps 

You forgot adding the old link to "redirects"

Let me know if in doubt ;)

Comment (DevStarOps) [reply rules/use-iapimarker-with-webapplicationfactory/rule.md:4]: Added, thanks 😊

File: rules/use-iapimarker-with-webapplicationfactory/rule.md [hunk 2]
```diff
@@ -30,10 +35,13 @@ This small change leads to a long road of pain:
 2. Which means you need to make your tests internal and
 3. In turn add an `InternalsVisibleTo` tag to your test project for the test runner to be able to access the tests.
 
-#### Option 2 - public partial program class
+### Option 2 - public partial program class
 
 ![](Using-public-partial-program-class.jpg)
```
Comment (tiagov8) [inline rules/use-iapimarker-with-webapplicationfactory/rule.md:40]: Hey @DevStarOps 

the correct syntax for images is:
```
::: bad
![Figure: Bad example - Using a public partial program class](Using-public-partial-program-class.jpg)
:::

Please fix this one and others ;)
```

Comment (DevStarOps) [reply rules/use-iapimarker-with-webapplicationfactory/rule.md]: fixed 😊

**Review by tiagov8 — APPROVED** [review]

**Pull request merged**